src/components/SpecialResolution/SpecialResolutionSummary.vue

```diff
@@ -64,32 +37,37 @@
           </v-col>
 
           <v-col cols="8">
-            <span class="info-text">Changing from a {{ associationTypeToDescription(originalAssociationType) }}</span>
-            &nbsp;
-            <span class="info-text">to a {{ associationTypeToDescription(getAssociationType) }}</span>
+            <span class="info-text">{{ associationTypeToDescription(getAssociationType) }}</span>
           </v-col>
         </v-row>
       </div>
     </template>
 
+    <!-- Rules and Memorandum file name add here-->
+
+    <!-- Resolution summary section -->
+    <template>
```

Comment:
Don't need template block unless you need a v-if or want to encapsulate several components (just to keep them together for code folding).